static_test_driver/static_test_driver.ino

```diff
@@ -18,14 +18,19 @@ float chamber_temp, inlet_temp, outlet_temp,pressure,x,y,z;
 bool temp_status = false;
 #define CONFIGURATION DEMO
 
-//Thermocouple Setup
+
 #if CONFIGURATION == MK_2
 #define MAXDO1   3
 #define MAXCS1   4
 #define MAXCLK1  5
 Adafruit_MAX31855 Chamber_Thermocouple(MAXCLK1, MAXCS1, MAXDO1);
+//Pressure Setup
+#define PRESSURE_CALIBRATION_FACTOR 246.58
+#define PRESSURE_OFFSET 118.33
+#define PRESSURE_PIN 1
 #endif
 
+//Thermocouple Setup
```

Comment:
This comment is down here, but you are also setting up thermocouple stuff earlier for MK_2.  Just try to pay attention to this sort of thing organizationally - even though one thermocouple is configuration specific, still try to put all the code for handling the same kinds of sensors next to each other.